### ISSUE TITLE
AP / AP90 - review pending 160 entries

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/CORRECTIONS#439]
Details at https://github.com/sanskrit-lexicon/CORRECTIONS/issues/336#issuecomment-279876349

**Comment by Andhrabharati** [comment]
Just browsed through the list (<a href="https://gist.github.com/funderburkjim/2a0fe9be37217380c01d45a735791da4"> ap90_aphw2_short.txt</a> ) given by @funderburkjim at the said issue.

There are not just the 160 ? marked entries that are pending to be checked/reviewed; but even the other 4 types of ! marked entries as well-- I've seen couple of entries that need attention/correction in these.

**Comment by gasyoun** [comment]
> There are not just the 160 ?

Right, but can you close the small list before the big one? Or does not makes sense, as already fixed in the big one @Andhrabharati ?

**Comment by Andhrabharati** [comment]
I was just looking at the "pending" open issues.

Have no intention to compare AP90 and AP57, as AP57 is not "public".
